plugins/package-managers/pub/src/funTest/assets/projects/synthetic/pub-expected-output-multi-module.yml

```diff
@@ -61,10 +61,10 @@ packages:
       value: ""
       algorithm: ""
   source_artifact:
-    url: ""
+    url: "https://pub.dev/packages/collection/versions/1.16.0.tar.gz"
```

Comment:
Could you explain in the commit message what a "hosted packages" is? I.e., how does it compare to "non-hosted (regular?) packages"? If there is something like the latter, are these not supported? And why?